code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -1,22 +1,32 @@
 /mob/living/simple_animal/hostile/netherworld
 	name = "creature"
 	desc = "A sanity-destroying otherthing from the netherworld."
-	icon_state = "otherthing"
-	icon_living = "otherthing"
-	icon_dead = "otherthing-dead"
-	health = 80
-	maxHealth = 80
+	icon_state = "otherthing-pink"
+	icon_living = "otherthing-pink"
+	icon_dead = "otherthing-pink-dead"
+	health = 125
+	maxHealth = 125
 	obj_damage = 100
 	melee_damage_lower = 25
-	melee_damage_upper = 50
-	attacktext = "slashes"
+	melee_damage_upper = 30
+	attacktext = "chomps"
 	attack_sound = 'sound/weapons/bladeslice.ogg'
-	faction = list("creature")
 	speak_emote = list("screams")
 	gold_core_spawnable = CHEM_MOB_SPAWN_HOSTILE
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
-	faction = list("nether")
+	faction = list("nether", "mining", "cult")
+	vision_range = 2
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	unsuitable_atmos_damage = 15
+	weather_immunities = list("lava","ash")
```

Comment:
Understandable but what kind of design problem you see with them having no immunities? Explain.

Reply:
I wanted to make them more in line with the other mining mobs.